--- vim-state.coffee	2016-07-15 14:32:13.339154889 +0200
+++ myversion.coffee	2016-07-15 14:34:53.363301891 +0200
@@ -98,6 +98,7 @@
       'toggle-case-now': => new Operators.ToggleCase(@editor, this, complete: true)
       'yank': => @linewiseAliasedOperator(Operators.Yank)
       'yank-line': => [new Operators.Yank(@editor, this), new Motions.MoveToRelativeLine(@editor, this)]
+      'yank-to-last-character-of-line': => [new Operators.Yank(@editor, this), new Motions.MoveToLastCharacterOfLine(@editor, this)]
       'put-before': => new Operators.Put(@editor, this, location: 'before')
       'put-after': => new Operators.Put(@editor, this, location: 'after')
       'join': => new Operators.Join(@editor, this)
